CalSmokeApp/bin/make/ipa-cal.sh

```diff
@@ -133,7 +133,7 @@ fi
 cat >"${XTC_DIR}/Gemfile" <<EOF
 source "https://rubygems.org"
 
-gem "calabash-cucumber", "2.99.0"
+gem "calabash-cucumber"
 gem "json", "1.8.6"
```

Comment:
No.  This needs to be "calabash-cucumber", "2.99.0"

We can use cucumber 3.x locally, but in Test Cloud we must use 2.x

Reply:
But the latest version of calabash-cucumber gem is 0.21.10
https://rubygems.org/gems/calabash-cucumber/versions/0.21.10

Reply:
Apologies, I continue to make this mistake - confusing `calabash-cucumber` and `cucumber`.